docs/cli-usage.md

```diff
@@ -190,5 +190,42 @@ The best way to find out how that works is playing around with different values
 
 !!! note
 
-    Some identifiers do have text transformation enabled by default. 
+    Some identifiers do have text transformation enabled by default.
     Nevertheless make sure to **explicitly set** text transformation styles if you need them! All text transformation may be disabled by default in a future release of the generator.
+
+### Default Identifier Style settings
+(Please note that this list is work in progress)
```

Comment:
IMHO this sentence does not give additional information on the quality of the data below and therefore does not add value and should be removed.

Either the documented default value is correct (which means it is not WIP), or the documented value is wrong (I'd prefer to not document it if we don't know exactly, instead of spreading mis-information), or the value is undocumented (which is self explanatory).

Additionally the whole docs are currently marked with a big "This is WIP" banner on top of each page, so readers should already be aware.

Reply:
yes , this should not be merged and if a mark for the review 
for some it is might be either right or wrong, but not checked so far which one applies

Reply:
Uh oh I didn't realize this is still WIP.

Reply:
pls remove the WIP marker now that it's done. :)

Reply:
```suggestion
```